fvm/state/accounts.go

```diff
@@ -41,6 +42,34 @@ func NewAccounts(stateHolder *StateHolder) *Accounts {
 	}
 }
 
+func (a *Accounts) AllocateStorageAddress(address flow.Address) (uint64, error) {
+	var index uint64
+	indexBytes, err := a.getValue(address, false, KeyStorageIndex)
+	if err != nil {
+		return 0, err
+	}
+
+	if len(indexBytes) == 0 {
+		// if not exist for the first time set it to zero and return
+		index = 0
+	} else if len(indexBytes) != 8 {
+		// this should be fatal
+		return 0, fmt.Errorf("invalid storage index byte size (%d != 8)", len(indexBytes))
```

Comment:
Maybe refactor the magic number 8 into a constant, e.g. `indexValueSize`

Reply:
fixed